src/main/kotlin/no/nav/familie/ba/sak/behandling/BehandlingService.kt

```diff
@@ -97,6 +98,28 @@ class BehandlingService(private val behandlingRepository: BehandlingRepository,
                         behandlingId)
             }
 
+    fun hentDataForKonsistensavstemming(): List<PerioderForBehandling> {
+        val relevanteBehandlinger = behandlingRepository.finnSisteIverksatteBehandlingFraLøpendeFagsaker()
+        return relevanteBehandlinger
+                .map { behandlingId ->
+                    val andelerIRelevantBehandling = beregningService.hentAndelerTilkjentYtelseForBehandling(behandlingId)
```

Comment:
Denne koden kommer kalle databasen 1 gang for hver behandling som er aktuell å hente data for. 
Det er mulig denne ikke skal være koden som skal kjøre når dere har 600k avstemminger, men hvis det kanskje er det så skulle jeg anbefale å endre den til å ikke gjøre 1 kall for hver behandling. 

Tror det er mulig med en liten endring her og en endring i beregningService
```suggestion
        return relevanteBehandlinger.chunked(1000)
                .map { behandlingIder ->
                    val andelerIRelevantBehandling = beregningService.hentAndelerTilkjentYtelseForBehandlinger(behandlingIder)
```

Reply:
lurt!